lib/routes/admin-api/feature.js

```diff
@@ -159,12 +157,12 @@ class FeatureController extends Controller {
             const feature = await this.featureToggleStore.getFeature(
                 featureName,
             );
-
             feature[field] = value;
+            const validFeature = await featureShema.validateAsync(feature);
```

Comment:
Note: make sure we only update fields we are allowed to update with the help of the featureSchema.